Updated modules patch 1

lilleannette opened the pull request.

lilleannette committed:
Update srjob.sh

lilleannette committed:
Update extract_clim_iceh.sh

lilleannette committed:
Update extract_clim_iceh_update.sh

lilleannette committed:
Delete bin/ice_climatology/extract_clim_iceh.sh

lilleannette committed:
Rename extract_clim_iceh_update.sh to extract_clim_iceh.sh

lilleannette committed:
Update compile_model.sh

lilleannette committed:
Delete TP0a1.00/expt_01.0/pbsjob.sh

lilleannette committed:
Update create_ref_case.sh

Jipingnersc reviewed (commented):
During my test, extract_clim_iceh.sh was blocked at Line 87 due to missing the module of 
CDO/1.9.10-iimpi-2022a. Then loading another candidate module as follows:
ml load CDO/2.0.6-gompi-2022a

In the end, ice_initial.nc is obtained as expected.

Jipingnersc committed:
Update extract_clim_iceh.sh

Due to missing the module of CDO/1.9.10-iimpi-2022a
, it was replaced by CDO/2.0.6-gompi-2022a
